core/modules/update_ttl.h

```diff
@@ -0,0 +1,12 @@
+#ifndef BESS_MODULES_UPDATE_TTL_H_
+#define BESS_MODULES_UPDATE_TTL_H_
+
+#include "../module.h"
+
+// Swap source and destination IP addresses and UDP/TCP ports

```

Comment:
Please update the class comment to reflect the module's behavior and usage.

Also, if this module is responsible for dropping TTL <= 1 packets (as it appears to be), then you should describe that as well.